Source/WebKit/WebProcess/Network/webrtc/LibWebRTCNetwork.cpp

```diff
@@ -37,11 +38,29 @@ namespace WebKit {
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(LibWebRTCNetwork);
 
+LibWebRTCNetwork::LibWebRTCNetwork(WebProcess& webProcess)
+#if ENABLE(WEB_RTC)
+    : m_mdnsRegister(*this)
+#endif
+    , m_webProcess(webProcess)
```

Comment:
This won't build if `ENABLE(WEB_RTC)` is false.

Reply:
Just make `m_webProcess` the first data member to avoid the issue I guess.

Reply:
Oh that's a good point. Definitely saved me from waiting for the bots. Thank you and fixed :)